packages/footer/src/styledComponents/Footer.js

```diff
@@ -0,0 +1,31 @@
+import styled from "styled-components";
+import { colors as colorConstants } from "@crave/farmblocks-theme";
+
+const margin = "16px";
+
+const FooterContainer = styled.div`
+  font-family: Lato, sans-serif;
```

Comment:
we need to remember to remove this when/if we replace the a link with a future Link component

Reply:
Created https://github.com/CraveFood/farmblocks/issues/121 to address this